src/vivarium_gates_iv_iron/data/loader.py

```diff
@@ -77,11 +78,12 @@ def get_data(lookup_key: str, location: str) -> pd.DataFrame:
         data_keys.PREGNANCY_OUTCOMES.OTHER: load_pregnancy_outcome,
         data_keys.MATERNAL_DISORDERS.CSMR: load_standard_data,
         data_keys.MATERNAL_DISORDERS.INCIDENCE_RATE: load_standard_data,
-        #data_keys.MATERNAL_DISORDERS.DISABILITY_WEIGHT: load_maternal_disorders_disability_weight,
```

Comment:
this YLDS key is replacing the DISABILITY_WEIGHT key, right? Might as well delete this rather than commenting it.